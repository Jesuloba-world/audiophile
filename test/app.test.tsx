import TestRenderer from "react-test-renderer";
import MyApp from "pages/_app";

const dummyComponent = () => <div>For testing purposes</div>;

describe("test _app", () => {
	it("Renders correctly", () => {
		const tree = TestRenderer.create(
			<MyApp pageProps={{}} Component={dummyComponent} />
		).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
