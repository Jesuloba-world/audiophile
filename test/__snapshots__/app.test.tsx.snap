// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test _app Renders correctly 1`] = `
<div>
  For testing purposes
</div>
`;
